wgsl/index.bs

```diff
@@ -4190,6 +4197,46 @@ Example:  `let minint = -2147483648;` is analyzed as follows:
     [=AbstractInt=] -2147483648 value is converted to the [=i32=] value -2147483648.
 * The result is that `minint` is declared to be the i32 value -2147483648.
 
+### `override` Expressions ### {#override-expr}
+
+Expressions that are evaluated at [=pipeline creation=] time are called <dfn
+noexport>override-expressions</dfn>.
+In order for an expression to be evaluated at pipeline creation time all
+[=identifiers=] used by the expression must [=resolve=] to:
+* [=override-declarations=], or
+* [=const-declarations=], or
+* [=const-functions=], or 
+* [=type aliases=], or 
+* [=structure=] names
+
+Override-expressions are a superset of [=const-expressions=], except that they
```

Comment:
I'd change this to a note:

> Note:  Each const-expression is also an override-expression.

But now the "must be concrete" rule seems out of place. It's really the use of such an expr as the initializer for an override-declaration that forces the materialization to a concrete type.

The rule is already expressed in override-declaration, so I'd leave it at that.